packages/plugin/src/protobufts-plugin.ts

```diff
@@ -94,6 +94,9 @@ export class ProtobuftsPlugin extends PluginBase {
                 "the default behaviour, this option has no effect.",
             excludes: ['eslint_disable'],
         },
+        disable_service_types: {
+            description: 'Prevents the generation of service metadata.',
+        },
```

Comment:
There is a caveat to this option: The service metadata is used by generated clients, and the only way to disable clients completely is to set the option `force_client_none`. `client_none` is not enough. The user could have `(ts.client) = GRPC1_CLIENT` in a proto file.

Generating code that cannot compile could be considered a bug, so I wonder how we could avoid that.

Maybe the option should be named `force_disable_services`, and do the same as `force_client_none` and `force_server_none`. The option definition could use an `excludes` array to automatically disallow options like `client_grpc1` that do not make sense to be set at the same time.

Reply:
@ColinLaws, what do you think about this caveat, and the proposed fix?

Reply:
Sorry for leaving this pull request Hanging @timostamm My day job working on a project that is dependent on your package is taking up nearly all of my time. 

I see the issue you're mentioning, and I do agree with your suggested solution to resolving this. If someone is consuming this package and does not wish to generate the services, then I agree it's a good bet that they aren't going to write code that is dependent on them. I can try implementing a fix for this likely sometime this week. I may be able to fit it into what I'm doing at work, as this is the only package keeping us from upgrading to NG13.

Reply:
Thank you @ColinLaws, looking forward to it :)